### ISSUE TITLE
ValueError thrown if label value is too high

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#2607]
### Description

When trying to run the function in lumbar FOV, if specifying a starting point at label =19 (corresponding to T11-T12 disc), the code crashes with a ValueError. The reason being that the labels on the template are not going high enough.

### Steps to Reproduce

~~~
# here we use sct_testing_data/t2.nii.gz image, even if it is centered in the cervical cord, but the crash is independent from the image content anyways.
cd sct_testing_data/t2
sct_label_vertebrae -i t2.nii.gz -s t2_seg.nii.gz -c t2 -initz 25,19
...
Current disc: 19 (z=50). Direction: superior
.. Peak found: z=0 (correlation = 0.24951264940910967)
.. Switching to inferior direction.
Traceback (most recent call last):
  File "/Users/julien/code/sct/scripts/sct_label_vertebrae.py", line 427, in <module>
    main()
  File "/Users/julien/code/sct/scripts/sct_label_vertebrae.py", line 372, in main
    verbose=verbose, path_template=path_template, path_output=path_output, scale_dist=scale_dist)
  File "/Users/julien/code/sct/spinalcordtoolbox/vertebrae/core.py", line 222, in vertebral_detection
    current_z = init_disc[0] - list_distance[list_disc_value_template.index(current_disc)]
ValueError: 20 is not in list
~~~

### Possible solution

Increase the number of labels from the template.

**Comment by saulo-p** [comment]
I could not replicate the issue with the parameters above.
From the history, the commit that I referenced above seems to have fixed the edge case (I checked out to the previous commit and then I was able to replicate).

@jcohenadad  I did not understand if your **Possible solution** suggests that more entries need to be added to the respective `info_labels.txt`, but as far as the bug is concerned, I think the issue can be closed.

**Comment by lrouhier** [comment]
I was able to replicate the error. 
sct version: 
Spinal Cord Toolbox (git-master-ca92ce39b85716419527b8ccab9819bd3c99f484) 

I looked into it a bit and I think that [these lines](https://github.com/neuropoly/spinalcordtoolbox/blob/ca92ce39b85716419527b8ccab9819bd3c99f484/spinalcordtoolbox/vertebrae/core.py#L210-L213) do not work. 
The reason for that would be that                
` approx_distance_to_next_disc = list_distance[list_disc_value_template.index(current_disc)]` throws a `ValueError` and not an `IndexError` as suggested. I will look into it but it does seems to fix this, I'll open a PR if I think that this will fix this. 

This avoids modifying our current template.
